src/AcmExpandable/AcmExpandableCard/AcmExpandableCard.tsx

```diff
@@ -19,20 +19,32 @@ const useStyles = makeStyles({
     },
 })
 
+const onToggle = (acmCardID: string, open: boolean, setOpen: (open: boolean) => void) => {
+    setOpen(!open)
+    if (localStorage.getItem(acmCardID) === 'show') {
```

Comment:
does localstorage let you set a boolean, if so we could do that instead of reading between two string values?

Reply:
No, it expects a string there

Reply:
Sadness =/